fv3core/pace/fv3core/initialization/geos_wrapper.py

```diff
@@ -0,0 +1,256 @@
+from datetime import timedelta
+from typing import Dict
+
+import f90nml
+import numpy as np
+
+import pace.util
+from pace import fv3core
+
+
+class GeosDycoreWrapper:
+    """
+    Provides an interface for the Geos model to access the Pace dycore.
+    Takes numpy arrays as inputs, returns a dictionary of numpy arrays as outputs
```

Comment:
So this only works for numpy and cpu backend? We should extend this to take cupy array as well?

Reply:
Ideally we can also make this work on GPU. The reason for the current input/output setup is to comport with Purnendu's code, which puts the Fortran data in numpy arrays and reads a dictionary back out.

Reply:
Should be gpu-functional now, I hope?